pkg/autodiscovery/autoconfig.go

```diff
@@ -540,3 +628,47 @@ func GetConfigErrors() map[string]string {
 func GetResolveWarnings() map[string][]string {
 	return errorStats.getResolveWarnings()
 }
+
+// processNewService takes a service, tries to match it against templates and
+// triggers scheduling events if it finds a valid config for it.
+func (ac *AutoConfig) processNewService(svc listeners.Service) {
+	// in any case, register the service and store its tag hash
+	ac.store.setServiceForEntity(svc, svc.GetEntity())
+	// todo store?
```

Comment:
what's this comment for?

Reply:
probably an old comment for putting the template cache in the store, removing